go/internal/feast/repoconfig.go

```diff
@@ -51,30 +51,29 @@ func NewRepoConfigFromJson(repoPath, configJSON string) (*RepoConfig, error) {
 	return &config, nil
 }
 
+// NewRepoConfigFromFile reads the `feature_store.yaml` file in the repo path and converts it
+// into a RepoConfig struct.
 func NewRepoConfigFromFile(repoPath string) (*RepoConfig, error) {
 	data, err := os.ReadFile(filepath.Join(repoPath, "feature_store.yaml"))
+	if err != nil {
+		return nil, err
+	}
 	repoPath, err = filepath.Abs(repoPath)
 	if err != nil {
 		return nil, err
 	}
 
 	config := RepoConfig{}
-	if err != nil {
-		return nil, errors.New(fmt.Sprintf("FEAST_REPO_PATH: %s not found", repoPath))
-	} else {
-		if err = yaml.Unmarshal(data, &config); err != nil {
-			return nil, err
-		}
+	if err = yaml.Unmarshal(data, &config); err != nil {
+		return nil, err
 	}
-
 	config.RepoPath = repoPath
 	return &config, nil
 }
 
 func (r *RepoConfig) GetRegistryConfig() *RegistryConfig {
```

Comment:
Is the automatic refresh going to happen in a different PR?

Reply:
yes